pkg/formatters/summaryFormatter_test.go

```diff
@@ -141,3 +143,91 @@ func TestSummaryFormatterMultipleRunsDifferentResultsProducesExpectedTotalsCount
 			"Total:9 Passed:1 PassedWithDefects:1 Failed:2 EnvFail:2 UNKNOWN:1 Active:1 Ignored:1\n"
 	assert.Equal(t, expectedFormattedOutput, actualFormattedOutput)
 }
+
+func TestSummaryFormatterMultipleRunsWithLostRunsDoesNotDisplayLostRunsAndCountsThem(t *testing.T) {
```

Comment:
Change these as well please. Something like this may be more readable:
TestSummaryFormatterWithMultipleRunsReturnsOnlyFinishesRuns